library/std/src/os/vxworks/mod.rs

```diff
@@ -1,6 +1,7 @@
 //! VxWorks-specific definitions
 
 #![stable(feature = "raw_ext", since = "1.1.0")]
+#![deny(unsafe_op_in_unsafe_fn)]
```

Comment:
You can remove the `unsafe_op_in_unsafe_fn` attribute from `fs.rs` and `raw.rs`, since they are within this module.

Also, can you upgrade these from `deny` to `forbid`?

Reply:
Done.